src/main/java/frc/robot/robotcontainers/NewRobotContainer.java

```diff
@@ -30,6 +34,7 @@ public class NewRobotContainer extends RobotContainer {
   private final CommandXboxController operator = new CommandXboxController(1);
 
   private final SwerveSubsystem s_Swerve = SwerveSubsystem.getInstance();
+  private final IntakeSubsystem intake = IntakeSubsystem.getInstance();
```

Comment:
You can remove this line. intake subsystem will be created when the command is scheduled first time.